### ISSUE TITLE
S1 - Fine Tune Profit LLM

### ISSUE THREAD

**Issue by yu-jeffy** [yu-jeffy/profit-llm-morality#11]

**Comment by yu-jeffy** [comment]
trained for 18 epochs, 2e-4 learning rate

it works! does not abide by ethical guardrails and answer in financial interest, even in morally skewed prompts

**Comment by yu-jeffy** [comment]
may want to train for more epochs to create a version that is even further aligned with our dataset, but the 18 epoch version will work for now
